src/controlPane.js

```diff
@@ -0,0 +1,10 @@
+import express from 'express';
+
+function startControlPane(pool, port) {
```

Comment:
one alternative would be, if `startControlPane` took `control` as an argument - then you could pass an existing server to it, and consumers could do `startControlPane(express(), pool, port)`?